drivers/crypto/crypto_mtls_shim.c

```diff
@@ -48,11 +48,13 @@ struct mtls_shim_session mtls_sessions[CRYPTO_MAX_SESSION];
 #error "You need to define MBEDTLS_MEMORY_BUFFER_ALLOC_C"
 #endif /* MBEDTLS_MEMORY_BUFFER_ALLOC_C */
 
+#define mtls_get_ctx(c, m) \
```

Comment:
if you make a macro, make it in capital letters.

Reply:
This is part of the other PR #20434. Will fix there.

Reply:
sure

Reply:
@tbursztyka Done. Please re-review.